File: c/divisiblity_5_11.c
#include <stdio.h>

int main()
{
    int inte;
    printf("Tell me a Number!\n");
    scanf("%d",&inte);
    if (inte % 5 == 0 && inte % 11 == 0)
    {
        printf("%d is divisible by both 5 and 11!",inte);
    }
    else if (inte % 5 == 0)
    {
        printf("%d is divisible by 5!",inte);
    }
    else if (inte % 11 == 0)
    {
        printf("%d is divisible by 11!",inte);
    }
    else
    {
        printf("Not divisible by any!");
    }
}

File: python/divisiblity_5_11.py
inte=int(input("Tell me a Number: "))
if inte % 5 == 0 and inte % 11 == 0:
    print(inte,"is divisible by both 5 and 11!")
elif inte % 5 == 0:
    print(inte,"is divisible by only 5!")
elif inte % 11 == 0:
    print(inte,"is divisible by only 11!")
else:
    print("Not disible by Any!")